sources/web/datalab/polymer/components/directory-picker-dialog/directory-picker-dialog.ts

```diff
@@ -20,6 +20,7 @@
  */
 interface DirectoryPickerDialogCloseResult extends BaseDialogCloseResult {
```

Comment:
I'd like to keep this name until we actually have a full file picker. Right now, we still can't pick files, as in, you can't really select a file and click a button to confirm, you're still only selecting the current open directory, but now you have an additional input box.